android/app/src/androidTest/java/com/zingo/ExportTest.kt

```diff
@@ -0,0 +1,19 @@
+package com.zingo
+
+// import android.content.Context
+import androidx.test.filters.SmallTest
+// import androidx.test.runner.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+// import org.junit.runner.RunWith
+
+@SmallTest
+class ExportTest {
+    @Test
```

Comment:
here you can put this:
`private external fun execute(cmd: String, args: String): String`

Reply:
so you can do this:
`val export = execute("export", "")`